pmm-tests/tls-ssl-setup/mongodb/mongodb_ssl_setup.sh

```diff
@@ -1,48 +1,47 @@
 #!/bin/sh
 
-
 while [ $# -gt 0 ]; do
-
    if [[ $1 == *"--"* ]]; then
         param="${1/--/}"
         declare $param="$2"
    fi
-
   shift
 done
 
-if [ -z "$mongodb_version" ]
-then
-      export mongodb_version=4.4
+if [ -z "$mongodb_version" ]; then
+  export mongodb_version=4.4
```

Comment:
This ticket is about adding 7.0 support to our scripts. I would let the defaults bumping into a separate task since there are 2 questions: 1. Why 6.0 and not 7.0? 2. What about Postgres and MySQL(it make sense to bump altogether).

Reply:
well, yes, makes sense. approving